app-editors/vis/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>contact@hacktivis.me</email>
+</maintainer>
+<maintainer type="project">
+	<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
+</maintainer>
+<longdescription lang="en">
+Vis aims to be a modern, legacy free, simple yet efficient editor combining the strengths of both vi(m) and sam.
+</longdescription>
+<use>
+	<flag name="tre">more memory efficient regex search using <pkg>dev-libs/tre</pkg></flag>
+	<flag name="curses">use curses (not only GNU ncurses), else it can cause flickering</flag>
```

Comment:
You want to use USE=ncurses here. There's not a single package using USE=curses, nor really a supported alternative curses implementation in Gentoo. Using USE=curses only forces most of the users to set the flag manually which is pointlessly inconvenient.